test/disabled_checks_test.rb

```diff
@@ -0,0 +1,150 @@
+# frozen_string_literal: true
+require "test_helper"
+
+class DisabledChecksTest < Minitest::Test
+  def setup
+    @tracer_one = TracerCheck.new
+    @tracer_two = TracerCheck.new
+    @tracer_one.stubs(:code_name).returns('TracerOneCheck')
```

Comment:
Instead of stubbing you could create two real classes nested under `DisabledChecksTest`:

```
class TrackerOne < TracerCheck; end
class TrackerTwo < TracerCheck; end
```

I feel it would avoid this test knowing about the internals for `Checks`.